templates_ark/example_neighborhood_analysis_script.ipynb

```diff
@@ -1,18 +1,32 @@
 {
```

Comment:
<p>What about adding a min_k as well? Did you ever look at <code>k</code> values below 5? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/714/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
Yes, I have looked at k values below 5 when looking at specific regions of the lymph node (i.e. what are the neighborhoods only within the T cell zone?). But I can add a min_k anyways for flexibility.